src/pages/costModels/costModelsDetails/components/costModelsDetailsToolbar.tsx

```diff
@@ -332,6 +332,11 @@ class CostModelsDetailsToolbarStateful extends React.Component<
                       children: t('toolbar.sources.secondary.aws'),
                       key: 'aws',
                     },
+                    {
+                      value: 'GCP',
+                      children: t('toolbar.sources.secondary.gcp'),
+                      key: 'gcp',
+                    },
```

Comment:
You can omit changes to cost models. 

I don't know that the backend APIs are ready to support this?

Reply:
Done